src/main/java/net/javadiscord/javabot/systems/help/dao/HelpAccountRepository.java

```diff
@@ -66,6 +68,40 @@ public Optional<HelpAccount> getByUserId(long userId) throws SQLException {
 		}
 	}
 
+	/**
+	 * Gets as many Accounts as specified.
+	 *
+	 * @param page    The page.
+	 * @param size    The amount of account to return.
+	 * @return A {@link List} containing the specified amount of {@link HelpAccount}s.
+	 * @throws SQLException If an error occurs.
+	 */
```

Comment:
I think this sounds a bit better and is also more similar to other Javadoc in this class
```suggestion
	/**
	 * Gets a specified amount of {@link HelpAccount}s.
	 *
	 * @param page    The page.
	 * @param size    The amount of {@link HelpAccount}s to return.
	 * @return A {@link List} containing the specified amount of {@link HelpAccount}s.
	 * @throws SQLException If an error occurs.
	 */
```